plugins/catalog-backend/src/service/router.ts

```diff
@@ -17,33 +17,36 @@
 import express from 'express';
 import Router from 'express-promise-router';
 import { Logger } from 'winston';
-import { addLocationSchema, ItemsCatalog, LocationsCatalog } from '../catalog';
+import {
+  addLocationSchema,
+  EntitiesCatalog,
+  LocationsCatalog,
+} from '../catalog';
 import { validateRequestBody } from './util';
 
 export interface RouterOptions {
-  itemsCatalog?: ItemsCatalog;
+  entitiesCatalog?: EntitiesCatalog;
   locationsCatalog?: LocationsCatalog;
   logger: Logger;
 }
 
 export async function createRouter(
   options: RouterOptions,
 ): Promise<express.Router> {
-  const { itemsCatalog, locationsCatalog } = options;
-  const logger = options.logger.child({ plugin: 'catalog' });
+  const { entitiesCatalog, locationsCatalog } = options;
   const router = Router();
 
-  if (itemsCatalog) {
-    // Components
+  if (entitiesCatalog) {
+    // Entities
     router
-      .get('/components', async (_req, res) => {
-        const components = await itemsCatalog.components();
-        res.status(200).send(components);
+      .get('/entities', async (_req, res) => {
+        const entities = await entitiesCatalog.entities();
+        res.status(200).send(entities);
       })
-      .get('/components/:id', async (req, res) => {
+      .get('/entities/:id', async (req, res) => {
```

Comment:
This is :name and not ID right? Also assume we want have have something like `/entities/:kind/:namespace/:name`?

Reply:
I'm not sure we should have this particular endpoint in this particular form at all :) for now it's only a remnant until I do the final fixes for #880. I'd like to leave this as is because it'll be expired or look completely different when we introduce the filterability we talked about.

Reply:
If we keep an iterated version of it I'd like it to be `/entities/:namespace/:name`, excluding the kind. Otherwise I'm sure we'll be repeating some of the old mistakes of the internal API.

Reply:
Ok, assuming you mean that  kind isn't part of the tuple used to uniquely identify an entity? It's not how k8s works but I don't think that discussion is in the scope of this PR :grin:, added a note in the architecture doc though